docs/src/getting_started.ipynb

```diff
@@ -51,13 +61,66 @@
     "import numpy as np"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "5e9bf8e3-07fd-4e4d-bdb8-d32a48a366c7",
+   "metadata": {},
+   "source": [
+    "## Part 1: Events and EventSets\n",
+    "\n",
+    "The most basic unit of data in Temporian is an **event**. An event consists of a timestamp and a set of feature values.\n",
+    "\n",
+    "Events are not handled individually. Instead, events are grouped together into an **`EventSet`**.\n",
+    "\n",
+    "`EventSets` are the main data structures in Temporian, and represent **[multivariate and multi-index time sequences](../user_guide/#what-is-temporal-data)**. Let's break that down:\n",
+    "\n",
+    "- \"multivariate\" indicates that each event in the time sequence holds several feature values.\n",
+    "- \"multi-index\" indicates that the events can represent hierarchical data, and be therefore grouped by one or more of their features' values.\n",
+    "- \"sequence\" indicates that the events are not necessarily sampled at a uniform rate (in which case we would call it a time \"series\").\n",
+    "\n",
+    "You can create an `EventSet` from a pandas DataFrame, NumPy arrays, CSV files, and more. Here is an example containing four events and three features (one of which is used as an `index`):"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "46ebad7f-f4b4-4850-bc12-8552e55a3f6b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "evset = tp.event_set(\n",
+    "    timestamps=[\"2023-02-04\", \"2023-02-06\", \"2023-02-07\", \"2023-02-07\"],\n",
+    "    features={\n",
+    "        \"feature_1\": [0.5, 0.6, np.nan, 0.9],\n",
+    "        \"feature_2\": [\"red\", \"blue\", \"red\", \"blue\"],\n",
+    "        \"feature_3\":  [10.0, -1.0, 5.0, 5.0],\n",
+    "    },\n",
+    "    indexes=[\"feature_2\"],\n",
+    ")\n",
+    "evset"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "effc4483-9a1a-4e21-b376-3ed188ced821",
+   "metadata": {},
+   "source": [
+    "An `EventSet` can hold one or several time sequences, depending on what its `index` is.\n",
+    "\n",
+    "If it has no index, it will hold a single multivariate time sequence, which means that all events will be considered part of the same group and will interact with each other when operators are applied.\n",
+    "\n",
+    "If it has one (or many) indexes, its events will be grouped by their `indexes` values, so it will hold one multivariate time sequence for each unique value (or unique combination of values) of its indexes, and operators will be applied to each time sequence independently.\n",
+    "\n",
```

Comment:
What about:

An EventSet can hold one or several time sequences, depending on its index.
- If it has no index, an EventSet holds a single multivariate time sequence.
- If it has one (or more) indexes, the events are grouped by their index values. This means that the EventSet will hold one multivariate time sequence for each unique value (or unique combination of values) of its indexes. 
Many operators are applied on each time sequence of an EventSet independently. Indexing is the primary way to handle rich and complex databases. For instance, in a retail database, you can index on customers.